gen/struct.go

```diff
@@ -19,8 +19,13 @@ func Struct(rt reflect.Type, gens map[string]gopter.Gen) gopter.Gen {
 	}
 	fieldGens := []gopter.Gen{}
 	fieldTypes := []reflect.Type{}
+	assignable := reflect.New(rt).Elem()
```

Comment:
Is this necessary?
After this `assignable` should be the same to `rt`

Reply:
I would need to defer to Diego for this since he is the original author.